src/box/memtx_tree.cc

```diff
@@ -1210,7 +1210,7 @@ memtx_tree_index_size(struct index *base)
 	memtx_tx_story_gc();
 	/* Substract invisible count. */
 	return memtx_tree_size(&index->tree) -
-	       memtx_tx_index_invisible_count(in_txn(), space, base);
```

Comment:
AFAIC, you removed all instances of `memtx_tx_index_invisible_count`. Can we drop it completely?